client/src/modules/MentorRegistry/components/MentorRegistryTableContainer.tsx

```diff
@@ -337,17 +384,51 @@ export const MentorRegistryTableContainer = ({
   };
 
   const handleClearAllButtonClick = () => {
-    setCombinedFilter({ preferredCourses: [], technicalMentoring: [], preselectedCourses: [] });
+    setCombinedFilter({
+      preferredCourses: [],
+      technicalMentoring: [],
+      preselectedCourses: [],
+      githubId: [],
+      cityName: [],
+    });
     setTagFilters([]);
   };
 
+  useAsync(async () => {
+    try {
+      const { content, total } = await mentorRegistryService.filterMentorRegistries({
+        pageSize: PAGINATION,
+        currentPage,
+        githubId: combinedFilter.githubId?.length ? (combinedFilter.githubId[0] as string) : undefined,
+        cityName: combinedFilter.cityName?.length ? (combinedFilter.cityName[0] as string) : undefined,
```

Comment:
Can something like this be used?
`combinedFilter.githubId?.[0] ?? undefined` or `combinedFilter.githubId?.[0] ?? undefined`